tests/std/tests/P0556R3_bit_integral_power_of_two_operations/test.cpp

```diff
@@ -80,8 +81,12 @@ constexpr bool test_bit_width() {
     assert(bit_width(numeric_limits<T>::max()) == digits);
     assert(bit_width(T{1}) == T{1});

```

Comment:
The assertions on right lines 80 and 82 should have their RHSes updated to say `0` and `1` instead of `T{0}` and `T{1}`.